README.adoc

```diff
@@ -58,4 +58,54 @@ can be used to mint a supply-pegged ERC-20: TBTC. For an in-depth explanation
 about the design, see link:docs/rfc/rfc-1.adoc[RFC 1: tBTCv2 Design].
 
 == The Mechanism
-// FIXME
+
+=== Wallet Lifecycle
+
+==== Wallet Creation
+
+We kick off the wallet creation mechanism in
+link:https://github.com/keep-network/keep-core/tree/main/solidity/ecdsa[keep-core/ecdsa]
+via `Wallets.requestNewWallet`, which verifies:
```

Comment:
I would rather refer to [`Bridge.requestNewWallet`](https://github.com/keep-network/tbtc-v2/blob/614e0844a8647a7468c383d6994fd789feed5274/solidity/contracts/bridge/Bridge.sol#L895-L899) because it is the external function that will be called by maintainers.

Reply:
[17a0705](https://github.com/keep-network/tbtc-v2/pull/326/commits/17a0705aec8a2038eff23d6313cedb3a9fd2e106)